hm_pyhelper/hardware_definitions.py

```diff
@@ -318,6 +348,8 @@ def is_raspberry_pi() -> bool:
     # RAKwireless Hotspot Miner
     'rak-v1': {
         'FRIENDLY': 'RAK Hotspot',
+        'CPU_ARCH': 'aarch64',
+        'BALENA_DEVICE_TYPE': 'raspberrypi3-64',
```

Comment:
Think this is pi 4

Reply:
checked yocto layers. There are differences in bundled firmware. I thought balena image will behave same as Rasbian. 
eg.
https://github.com/agherzan/meta-raspberrypi/blob/378d4b6e7ba64b6a9a701457cc3780fa896ba5dc/conf/machine/raspberrypi4-64.conf

Shall I make this field a list instead of value, we can list down known sbc models carried in the same vendor model. we can use first one as default. Then list all of them in balena.yml as supported devices for this particular repo. 
eg.
https://github.com/balenalabs/balena-sound/blob/master/balena.yml#L36